Patch from https://sourceforge.net/p/extundelete/tickets/5/ which reflects superseding of
i_dir_acl with i_size_high as per https://github.com/tytso/e2fsprogs/commit/578fcbfd2 in
e2fsprogs. Also /usr/include/ext2fs/ext2_fs.h on RHEL/CentOS 6 has a preprocessor define
for backward compatibility.

--- extundelete-0.2.4/src/insertionops.cc		2012-12-30 18:23:32.000000000 +0100
+++ extundelete-0.2.4/src/insertionops.cc.i_size_high	2018-07-14 15:46:32.608277911 +0200
@@ -33,7 +33,7 @@
   os << "File flags: " << inode.i_flags << std::endl;
   os << "File version (for NFS): " << inode.i_generation << std::endl;
   os << "File ACL: " << inode.i_file_acl << std::endl;
-  os << "Directory ACL: " << inode.i_dir_acl << std::endl;
+  os << "High 32bits of size: " << inode.i_size_high << std::endl;
   os << "Fragment address: " << inode.i_faddr << std::endl;
   os << "Direct blocks: ";
   for (int n = 0; n < EXT2_NDIR_BLOCKS; n++)
